src/containers/SearchProvider.js

```diff
@@ -0,0 +1,27 @@
+import { geojson } from '../data/geojson';
+
+class SearchProvider {
+  async search({ query }) {
+    let matches = geojson.features.filter(
+      feature =>
+        feature.geometry &&
+        feature.geometry.coordinates &&
+        feature.geometry.coordinates.length > 0 &&
```

Comment:
probably good to check we have at least 2 coordinates. Instead of 

` feature.geometry.coordinates.length > 0 `

use

`  feature.geometry.coordinates.length >= 2 `